src/imports/logger/ErrorsTransport.js

```diff
@@ -0,0 +1,73 @@
+//import Rollbar from 'rollbar'
+
+import * as Sentry from '@sentry/node'
+import Transport from 'winston-transport'
+import { SPLAT } from 'triple-beam'
+import { forEach } from 'lodash'
+import Config from '../../server/server.config'
+
+const { env, sentryDSN, version, network } = Config
+const logEnvAllowed = !['test', 'development'].includes(env)
+
+// let rollbar
+// if (logEnvAllowed && rollbarToken) {
+//   rollbar = new Rollbar({
+//     accessToken: rollbarToken,
+//     captureUncaught: true,
+//     captureUnhandledRejections: true,
+//     payload: {
+//       environment: process.env.NODE_ENV
+//     }
+//   })
+// }
+
+let sentryInitialized = false
+if (logEnvAllowed && sentryDSN) {
+  Sentry.init({
+    dsn: sentryDSN,
+    environment: env
+  })
+
+  Sentry.configureScope(scope => {
+    scope.setTag('appVersion', version)
+    scope.setTag('networkUsed', network)
+  })
+
+  sentryInitialized = true
+}
+
+class CustomTransport extends Transport {
+  // eslint-disable-next-line
+  constructor(opts) {
+    super(opts)
+  }
+
+  log(context) {
+    const { message: generalMessage, ...data } = context
```

Comment:
also we have to send user data if present.

logger is attached to express req object
auth middleware adds 'user' property to the req object

so we could pass user (or req) to the Sentry then call identify

Reply:
https://github.com/GoodDollar/GoodServer/blob/master/src/imports/logger.js#L85